### PR TITLE
preapre feautre doc

### PR DESCRIPTION
**Pull request by jjiwooLim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ The Liberty plugin provides the following tasks for your project:
 | [libertyStart](docs/libertyStart.md#libertystart-task) | Starts the Liberty server in a background process. |
 | [libertyStatus](docs/libertyStatus.md) | Checks to see if the Liberty server is running. |
 | [libertyStop](docs/libertyStop.md#libertystop-task) | Stops the Liberty server. |
+| [prepare-feature](docs/prepareFeature.md#prepareFeature-task) | Prepares a user feature installation onto Liberty runtime. |
```
Comment (cherylking) [inline README.md:104]: I think this should be `prepareFeature` not `prepare-feature`. 
Prepares a user feature for installation to the Liberty runtime.

File: docs/prepareFeature.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+## prepareFeature task
+---
+The `prepareFeature` task  generates `features.json` file for user features. The `features.json` file is JSON file that contains the information found within a feature's ESA manifest file. JSONs are a key requirement for the installation of any liberty features(s) from a maven repository.
```
Comment (cherylking) [inline docs/prepareFeature.md:3]: Capitalize Liberty and Maven.

File: docs/prepareFeature.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+## prepareFeature task
+---
+The `prepareFeature` task  generates `features.json` file for user features. The `features.json` file is JSON file that contains the information found within a feature's ESA manifest file. JSONs are a key requirement for the installation of any liberty features(s) from a maven repository. 
+
+
+In Open Liberty and WebSphere Liberty runtime versions 21.0.0.11 and above, this goal can prepare the user feature to generate the JSON file.
```
Comment (cherylking) [inline docs/prepareFeature.md:6]: goal --> task

File: docs/prepareFeature.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+</project>
+
+ ```
+
+2. Provide the maven coordinate of the custom made `features-bom` file:
+ ```xml
+apply plugin: 'liberty'
+
+dependencies {
+    featuresBom 'userTest.user.test.features:features-bom:21.0.0.11'
```
Comment (cherylking) [inline docs/prepareFeature.md:41]: Same comment as in other PR...I think using version 21.0.0.11 in these examples is confusing.

**Review by cherylking — CHANGES_REQUESTED** [review]

File: docs/prepareFeature.md (hunk 4 above)
Comment (jjiwooLim) [reply docs/prepareFeature.md]: Using 1.0 instead

**Review by cherylking — APPROVED** [review]

**Pull request merged**